pipelines/_ingredients/ApPipe.yaml

```diff
@@ -73,8 +79,6 @@ contracts:
       diaPipe.connections.ConnectionsClass(config=diaPipe).exposure.name
   - detectAndMeasure.connections.ConnectionsClass(config=detectAndMeasure).subtractedMeasuredExposure.name ==
       transformDiaSrcCat.connections.ConnectionsClass(config=transformDiaSrcCat).diffIm.name
-  - detectAndMeasure.connections.ConnectionsClass(config=detectAndMeasure).diaSources.name ==
-      transformDiaSrcCat.connections.ConnectionsClass(config=transformDiaSrcCat).diaSourceCat.name
```

Comment:
Could you add contracts that link `filterDiaSrcCat` to `detectAndMeasure`(?) and `transformDiaSrcCat`?

Reply:
Though given the discussion on RFC-987, perhaps you'd deliberately want the input to `transformDiaSrcCat` to be ambiguous...

Reply:
(Please disregard my previous post -- a modified version of `ApPipe` that excluded `filterDiaSrcCat` would also lose the contracts that apply to it, so there's no need to specifically support the possibility.)

Reply:
@kfindeisen I've modified the contract as you suggested and have also rearranged the tasks in their execution order. Please let me know if things look good now.

Reply:
Yes, looks good. I'll keep my nose out of somebody else's review now. 🙂